ojd_daps_skills/pipeline/skill_ner_mapping/hard_coded_mapper_formatting.py

```diff
@@ -0,0 +1,51 @@
+"""One of script to format manually labelled skill spans
+in format for prev_skills_lookup_sample.json
+"""
+from ojd_daps_skills.utils.text_cleaning import short_hash
+from ojd_daps_skills import bucket_name
+from ojd_daps_skills.getters.data_getters import (
+    get_s3_resource,
+    save_to_s3,
+    load_s3_data,
+)
+
+import pandas as pd
+
+if __name__ == "__main__":
+
+    s3 = get_s3_resource()
+
+    hard_coded_skills = (
+        load_s3_data(
+            s3,
+            bucket_name,
+            "escoe_extension/inputs/data/skill_mappings/hard_labelled_skills.csv",
```

Comment:
I noticed this csv has some unwanted tabs in it which was causing `hardcoded_ojo_esco_lookup.json` to have things like `{"match_skill": "picking orders for dispatching\t", "match_id": "ee5e2297-39cc-4ed6-89e5-d58016118a7f\t"}`. 
So I've replaced "hard_labelled_skills.csv" with a cleaned up version

Reply:
it only occured for "ojo_skill": "Pick our customer orders" and "ojo_skill": "pack our customer orders"

Reply:
I kept your original file in S3 but renamed it s3://open-jobs-lake/escoe_extension/inputs/data/skill_mappings/hard_labelled_skills_unclean.csv (please delete if you don't have a need for it)